app/scripts/modules/google/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -176,18 +176,43 @@ module.exports = angular.module('spinnaker.gce.serverGroupCommandBuilder.service
     function populateCustomMetadata(metadataItems, command) {
       // Hide metadata items in the wizard.
       if (metadataItems) {
+        let customUserData = '';
+        let customUserDataKeys = [];
         if (angular.isArray(metadataItems)) {
+          let customUserDataItem = metadataItems.find( function( metadataItem) {
```

Comment:
Spacing is off on the anonymous function. 
If you're feeling fancy, you could do this with an arrow function:
```
let customUserDataItem = metadataItems.find(metadataItem => metadataItem.key === 'customUserData');
```